invites.go

```diff
@@ -38,12 +38,23 @@ func (srv *groupsAPI) SendInvite(ctx context.Context, req *notesv1.SendInviteReq
 		if err != nil {
 			return nil, statusFromModelError(err)
 		}
+		inviteGroup, err := srv.groups.GenerateGroupInviteLink(ctx,
+			&models.OneGroupFilter{GroupID: req.GroupId},
+			&models.GenerateGroupInviteLinkPayload{
+				GeneratedByAccountID: invite.RecipientAccountID,
+				ValidUntil:           invite.ValidUntil,
+			}, token.AccountID)
+		if err != nil {
+			return nil, statusFromModelError(err)
+		}
```

Comment:
InviteLink is not the same as Invite!
We do not generate an InviteLink in SendInvite

Reply:
I have fixed the code